### PR TITLE
#612 Bottom toolbar has some visual issues

### PR DESCRIPTION
**Pull request by AgriyaDev5**
## PR description

Bottom toolbar has some visual issues

### CODE REVIEW

File: src/editor/components/seSpinInput.js [hunk 1]
```diff
@@ -83,6 +84,7 @@ export class SESpinInput extends HTMLElement {
     case 'src':
       this.$img.setAttribute('src', newValue);
       this.$label.remove();
+      this.$div.style.width = '82px';
```
Comment (jfhenon) [inline src/editor/components/seSpinInput.js:87]: I dont think we should hard code the width. We could instead
- put the width as an attribute of the web component
- define a global css variable --se-spin-input-width 
- ...

**Review by jfhenon — CHANGES_REQUESTED** [review]
see comment please.

File: src/editor/components/seSpinInput.js (hunk 1 above)
Comment (AgriyaDev5) [reply src/editor/components/seSpinInput.js]: modified

File: src/editor/components/seSpinInput.js [hunk 2]
```diff
@@ -8,6 +8,10 @@ template.innerHTML = `
     margin: 5px 1px;
     padding: 3px;
   }
+  div.imginside {
+    --se-spin-input-width: var(--global-se-spin-input-width);
+    width: var(--se-spin-input-width);
```
Comment (jfhenon) [inline src/editor/components/seSpinInput.js:13]: why do we need --se-spin-input-width and --global-se-spin-input-width ?? 
Also, you added the width parameter to the se-spin-input. 
I think we need to decide between defining a global variable and reuse it inside the component or adding a width attribute to se-spin-input.

Comment (AgriyaDev5) [reply src/editor/components/seSpinInput.js:13]: shall i use global variable or width attribute ?

**Review by jfhenon — APPROVED** [review]

**Pull request merged**